app/controllers/admin/contents_controller.rb

```diff
@@ -9,7 +9,8 @@ def edit
                               {name: I18n.t('admin.contents.edit.main_links'), preferences: [:menu_1, :menu_1_icon_name, :menu_2, :menu_2_icon_name, :menu_3, :menu_3_icon_name, :menu_4, :menu_4_icon_name, :menu_5, :menu_5_icon_name, :menu_6, :menu_6_icon_name, :menu_7, :menu_7_icon_name]},
                               {name: I18n.t('admin.contents.edit.footer_and_external_links'), preferences: [:footer_logo,
                                                              :footer_facebook_url, :footer_twitter_url, :footer_instagram_url, :footer_linkedin_url, :footer_googleplus_url, :footer_pinterest_url,
-                                                             :footer_email, :community_forum_url, :footer_links_md, :footer_about_url]}]
+                                                             :footer_email, :community_forum_url, :footer_links_md, :footer_about_url]},
+                              {name: I18n.t('admin.contents.edit.user_guide'), preferences: [:user_guide_link]}]
```

Comment:
This `#edit` is getting hairy and hard to read. Honestly, I find it hard to understand what you just did. What about splitting these preference sections into objects that return each of this individual hashes and then we simply merge them into an array here? that is an approach that served me well in the past. Something like (I haven't tried it, but it should work :trollface:):

```ruby
# app/models/preference_sections/user_guide_section.rb

class UserGuideSection
  def name
    I18n.t('admin.contents.edit.user_guide')
  end

  def  preferences
    [:user_guide_link]
  end
end
```

```ruby
# app/controllers/admin/contents_controller.rb
module Admin
  class ContentsController < Spree::Admin::BaseController
    def edit
      @preference_sections = preference_sections.map do |preference_section|
        { name: preference_section.name, preferences: preference_section.preferences }
      end
    end

    private
  
    # Returns an array with the instances of preference sections, which all respond to #name and #preferences.
    def preference_sections
      Dir["models/preference_sections/**/*.rb"].map do |filename|
        basename = File.basename(filename, '.rb')
        basename.camelize.constantize.new
      end
  end
end

This makes it really easy to add, remove and change sections/preferences. And keep adding them all the time (whether they should be preferences in the UI or config settings in ofn_install it's another topic).